setup/mysql.sql

```diff
@@ -145,10 +147,25 @@ INSERT INTO `vexim`.`users`
     smtp, pop, realname, type, admin
 )
 VALUES 
-(   '1', 'siteadmin', 'siteadmin', '$1$12345678$2lQK5REWxaFyGz.p/dos3/', '65535', '65535', '', '',  'SiteAdmin', 'site', '1'
+(   '1', 'siteadmin', 'siteadmin', '$6$uR.EiB1dj5rrvwMF$Qh5LgdjOZavKXwhi9IF0Yuzu7qxsG.dLTTB8e./55ZRNfBuZVLnfUSOEXa0oWT6174myO.WYkOy83HYWAKPbK/', '65535', '65535', '', '',  'SiteAdmin', 'site', '1'
```

Comment:
I wonder, maybe we can leave this password unchanged? The user should change it right after logging in for the first time, so assuming they configure Vexim prior to logging in, their updated password will be hashed more securely anyway.
For same reason, I suggest to only leave CLEAR (and DES and/or SHA, if logging in with these as the configured crypt method would not work) as commented-out alternatives. Or maybe even drop all of these alternatives from our file, since we are discouraging from using these algorithms for auth anyway.